game.js

```diff
@@ -251,6 +254,15 @@ class Game {
 		text(this.score, width / 2, LINE_SPACING)
 	}
 
+	drawTargetScore() {
+		textAlign(CENTER, CENTER)
+		textSize(30)
+		fill(255, 255, 250)
+		strokeWeight(4)
+		text('You need a total of ' + targetScore + ' many points to win.', width / 2, LINE_SPACING * 10)
+		text('Only ' + (targetScore - this.score) + ' points to go!', width / 2, LINE_SPACING * 11)
```

Comment:
I added a progress bar, maybe someone could create something more creative later. Possibly a piggy bank that fills up with money because you said the grade 1s liked monopoly and money.